src/popup/popup.less

```diff
@@ -49,5 +49,18 @@ body {
 
 .info {
     white-space: nowrap;
-    color: #000;
+}
+
+@media (prefers-color-scheme: dark) {
+    body {
+        background: black;
+        color: white;
+    }
+}
+
+@media (prefers-color-scheme: light) {
+    body {
+        background: white;
+        color: black;
+    }
```

Comment:
@maximbaz I did some research on how to support dark/light theme, how about this solution?

By the way, to test it on Firefox, one new entity is needed in `about:config`:

`ui.systemUsesDarkTheme=1`

Perhaps some other color properties can be moved inside there, but I dont a have a good eye for this kind of things ;-)

Reply:
I'm happy with this approach. I would like the theme colours to match the browserpass ones.